receiver/splunksearchapireceiver/go.mod

```diff
@@ -0,0 +1,89 @@
+module github.com/open-telemetry/opentelemtry-collector-contrib/receiver/splunksearchapireceiver
+
+go 1.22.7
+
+require (
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza v0.114.0
+	github.com/stretchr/testify v1.9.0
+	go.opentelemetry.io/collector/component v0.114.0
```

Comment:
this receiver is still using v0.114.0 for otel dependencies, is that intended?

Reply:
Nope, good catch